SWLOR.Game.Server/Service/Loot.cs

```diff
@@ -254,6 +254,7 @@ public static void ProcessCorpse()
         {
             var self = OBJECT_SELF;
             SetIsDestroyable(false);
+            RemoveEffectByTag(self, "AGGRO_AOE"); // Dead creatures don't need aggro!
```

Comment:
This should be done in the AI service. Loot processing is a really random place to remove an AOE

Reply:
to be fair, it is under `ProcessCorpse()`
to be balanced, I'll move it to crea_death_aft